package.json

```diff
@@ -2,20 +2,32 @@
   "name": "react-show-svg",
   "version": "0.0.1",
   "description": "React wrapper component for svg icons",
+  "author": "Josip Sokec <josip@sokec.net>",
   "main": "build/index.js",
-  "dependencies": {
-    "react": "^15.5.4",
-    "webpack": "^2.6.1",
-    "svg-to-jsx": "https://github.com/reactor-studio/svg-to-jsx"
-  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
+    "test --u": "jest --u",
     "start": "webpack --watch",
     "build": "webpack"
   },
-  "author": {
-    "name": "Josip Sokec",
-    "email": "josip@sokec.net"
+  "jest": {
+    "snapshotSerializers": [
+      "enzyme-to-json/serializer"
+    ],
+    "testMatch": [
+      "**/?(*.)(spec|test).js?(x)"
+    ],
+    "verbose": false,
+    "moduleDirectories": [
+      "node_modules",
+      "src"
+    ]
+  },
+  "dependencies": {
+    "react": "16.2.0",
```

Comment:
You need to move `react`, `react-dom` and `weepack` from dependencies to `peerDependencies`